### DISCUSSION TITLE
should ClassRegistry.register() be a public instead of internal ?

### DISCUSSION THREAD

**Discussion by YashodipD** [iTwin/itwinjs-core#6588] [Q&A] [answered]
It might help in registering individual entityClass.

**Comment by MichaelBelousov** [comment]
considering the public alternative, [registerModule](https://github.com/iTwin/itwinjs-core/blob/ef8cbea4dd0209bee709cb569c2f5065abe06ff5/core/backend/src/ClassRegistry.ts#L200), I don't see why we can't publish this.

There likely are code bases who want a class per file in which case having to namespace import everything is obnoxious and I see no boundaries to us supporting this reasonably

**Comment by MichaelBelousov** [comment]
@YashodipD would you consider making a pull request with this change?

**Reply by ben-polinsky** [reply, marked as answer]
https://github.com/iTwin/itwinjs-core/pull/6593
